sympy/printing/pretty/pretty.py

```diff
@@ -1154,10 +1154,16 @@ def _print_SingularityFunction(self, e):
             return self._print_Function(e)
 
     def _print_gamma(self, e):
+        from sympy import Function
+        from sympy.functions.special import gamma_functions
+        from sympy import factorial
         if self._use_unicode:
             pform = self._print(e.args[0])
             pform = prettyForm(*pform.parens())
-            pform = prettyForm(*pform.left(greek_unicode['Gamma']))
+            if e.rewrite(factorial) == factorial(e.args[0] - 1):
```

Comment:
This is a very inefficient test for the gamma function. Is it not possible to use something like `if e.func == gamma`?